packages/playwright/src/index.ts

```diff
@@ -448,6 +467,13 @@ const playwrightFixtures: Fixtures<TestFixtures, WorkerFixtures> = ({
       await request.dispose();
     }
   },
+
+  server: async ({ _mockingProxy }, use) => {
```

Comment:
psa, This is a name clash with one of the custom fixtures in our own tests. If we end up going with this name, we should probably rename that custom fixture.